Misc/NEWS.d/3.14.0b4.rst

```diff
@@ -75,7 +75,7 @@ to the HTML5 standard.
 * Multiple ``=`` between attribute name and value are no longer collapsed.
   E.g. ``<a foo==bar>`` produces attribute "foo" with value "=bar".
 
-* Whitespaces between the ``=`` separator and attribute name or value are no
+* [REVERTED] Whitespaces between the ``=`` separator and attribute name or value are no
```

Comment:
Is it fine? I do not want to remove this, this is a history, but we should add an indication that this change was not in the final release.